cms_delete_content/templates/delete_confirm.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <template id="delete_confirm" name="CMS default delete confirm">
```

Comment:
AFAIK a client-side template is only for the client while this can be used on both client and server side and I think it's easier to override it (as any other website template). In any way you'll have to load it in JS, so what's the gain here?